sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/CosmosPagedFluxOptions.java

```diff
@@ -57,4 +64,43 @@ public CosmosPagedFluxOptions setMaxItemCount(Integer maxItemCount) {
         this.maxItemCount = maxItemCount;
         return this;
     }
+
+    /**
+     * Gets the tracer provider
+     * @return tracerProvider
+     */
+    public TracerProvider getTracerProvider() {
+        return this.tracerProvider;
+    }
+
+    /**
+     * Gets the tracer span name
+     * @return tracerSpanName
+     */
+    public String getTracerSpanName() {
+        return tracerSpanName;
+    }
+
+    /**
+     * Gets the databaseId
+     * @return databaseId
+     */
+    public String getDatabaseId() {
+        return databaseId;
+    }
+
+    /**
+     * Gets the service end point
+     * @return serviceEndpoint
+     */
+    public String getServiceEndpoint() {
+        return serviceEndpoint;
+    }
+
+    public void setTracerInformation(TracerProvider tracerProvider, String tracerSpanName, String serviceEndpoint, String databaseId) {
+        this.databaseId = databaseId;
+        this.serviceEndpoint = serviceEndpoint;
+        this.tracerSpanName = tracerSpanName;
+        this.tracerProvider = tracerProvider;
+    }
```

Comment:
This feels out of place, but because it is implementation API it is fine.

Reply:
Yes this is  an implementation api.